shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/BaseRule.g4

```diff
@@ -141,166 +509,836 @@ primaryKey
     : PRIMARY? KEY
     ;
 
-// TODO comb expr
-expr
-    : expr logicalOperator expr
-    | notOperator_ expr
-    | LP_ expr RP_
-    | booleanPrimary
-    ;
-
 logicalOperator
     : OR | OR_ | AND | AND_
     ;
 
-notOperator_
-    : NOT | NOT_
+comparisonOperator
+    : EQ_ | GTE_ | GT_ | LTE_ | LT_ | NEQ_
     ;
 
-booleanPrimary
-    : booleanPrimary IS NOT? (TRUE | FALSE | UNKNOWN | NULL)
-    | booleanPrimary SAFE_EQ_ predicate
-    | booleanPrimary comparisonOperator predicate
-    | booleanPrimary comparisonOperator (ALL | ANY) subquery
-    | predicate
+
+cursor_name
+    : name
+	;
+
+a_expr
```

Comment:
What's the meaning of `a_expr`, `b_expr` and `c_expr`?

Reply:
pg's source code is so named and applied directly, in order to be consistent with the source code. It's just an expression.